main.go

```diff
@@ -15,29 +16,53 @@ var (
 	ids    map[string]int
 	nextId int
 
-	ignored = map[string]bool{
-		"C": true,
-	}
+	ignored         map[string]bool
 	ignoredPrefixes []string
 	onlyPrefixes    []string
 
-	ignoreStdlib   = flag.Bool("s", false, "ignore packages in the Go standard library")
-	delveGoroot    = flag.Bool("d", false, "show dependencies of packages in the Go standard library")
-	ignorePrefixes = flag.String("p", "", "a comma-separated list of prefixes to ignore")
-	ignorePackages = flag.String("i", "", "a comma-separated list of packages to ignore")
-	onlyPrefix     = flag.String("o", "", "a comma-separated list of prefixes to include")
-	tagList        = flag.String("tags", "", "a comma-separated list of build tags to consider satisified during the build")
-	horizontal     = flag.Bool("horizontal", false, "lay out the dependency graph horizontally instead of vertically")
-	includeTests   = flag.Bool("t", false, "include test packages")
-	maxLevel       = flag.Int("l", 256, "max level of go dependency graph")
+	ignoreStdlib   = new(bool)
```

Comment:
I think this could be achieved a better way. `main()` could call another function which takes a settings struct. Then the tests could just call that. Something like:

`Depgraph(o Options)`

where `Settings` is a struct type that contains all the options. Then `main` could just create an instance of it and set all the flags to point to that.

The tests would just call `Depgraph` directly.